### PR TITLE
Update links and missing info

### PR DESCRIPTION
**Pull request by kfklein15**
Signed-off-by: KFK <80272258+kfklein15@users.noreply.github.com>

## What does this change?

Module: Final Projects
Week(s): Intro

## Description

  Updated the link to the roles in Gitbook. I amended 3 volunteers to 2 and added their roles for clarity. And added on the stand-up that they should read the other people's stand-up too.

## Who needs to know about this?

 @MarianaVazquezCYF @AliciaE08 @ShaunCYFSA @caromelendezl @martaesn 

## Rendered Pages

-----
[View rendered docs/finalproject/intro.md](https://github.com/CodeYourFuture/syllabus/blob/kfklein15-patch-2/docs/finalproject/intro.md)

### CODE REVIEW

File: docs/finalproject/intro.md [hunk 1]
```diff
@@ -38,20 +38,20 @@ The final projects form the final module of the course, comprising four weeks fo
 Ideally, the project team would consist of the following:
 
 - Four CYF trainees;
```
Comment (SallyMcGrath) [inline docs/finalproject/intro.md]:
```suggestion
- Three or four CYF trainees;
```

File: docs/finalproject/intro.md [hunk 2]
```diff
@@ -38,20 +38,20 @@ The final projects form the final module of the course, comprising four weeks fo
...
-- A product owner.
+- Two volunteer mentors ([Tech Lead](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles/tech-lead) and [Product Manager](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles/product-manager), forming a [balanced team](https://www.youtube.com/watch?v=Z_Q4Q8rCVpU); and
+- A [product owner](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles/product-owner).
 
 The mentor group would include an engineer (responsible for unblocking technical/architectural issues, _not_ an extra developer), designer (responsible for user research and UI/UX) and product manager (responsible for prioritisation and alignment with the product owner’s goals). This gives an opportunity to bring in volunteers from the broader tech community, and exposes the trainees to the roles they could be collaborating with in employment.
 
-For more information on roles [click here](./roles.md)
+For more information on roles [click here](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles)
```
Comment (SallyMcGrath) [inline docs/finalproject/intro.md]:
```suggestion
[More information on volunteer roles](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles)
```

Let's always update these when we see them. Links must make sense out of context. [Success criterion 2.4.4](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html)

File: docs/finalproject/intro.md [hunk 3]
```diff
@@ -38,20 +38,20 @@ The final projects form the final module of the course, comprising four weeks fo
...
 
 ## Weekly plan
 
 We’re going to run the final projects in weekly “sprints”, planning out what we’re going to do as teams and as individuals. Each week will therefore look something like:
 
-- **Daily standup**: post a daily message in the team Slack channel to let the rest of the team know what you’re working on, what progress you’ve made (even, and perhaps _especially_, if it’s none so far) and share anything that’s blocking you. Set a time for this and stick to it. Afterwards, post a team summary in the main class channel.
+- **Daily standup**: post a daily message in the team Slack channel to let the rest of the team know what you’re working on, what progress you’ve made (even, and perhaps _especially_, if it’s none so far) and share anything that’s blocking you. Set a time for this and stick to it. Don't forget to also read your team members updates and try to help them if they are blocked. Afterwards, post a team summary in the main class channel.
```
Comment (SallyMcGrath) [inline docs/finalproject/intro.md]:
```suggestion
- **Daily standup**: post a daily message in the team Slack channel to let the rest of the team know what you’re working on, what progress you’ve made (even, and perhaps _especially_, if it’s none so far) and share anything that’s blocking you. Set a time for this and stick to it. Don't forget to also read your team members updates and try to help them when they are blocked. Afterwards, post a team summary in the main class channel. 
```

Small tweak (BTW you can accept suggestions like this directly on the review page)

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
Tiny change and then this can go out too. Thanks so much!

**Comment by kfklein15** [comment]
@SallyMcGrath , I committed all your changes. Thank you so much for reviewing this :) Now you just have to approve it again, I think?

**Review by SallyMcGrath — APPROVED** [review]
LGTM 🎉

**Pull request merged**